openshift/ansible/__init__.py

```diff
@@ -1,14 +1,19 @@
-import copy
 import json
 import re
 import string
 
 from ansible.module_utils.basic import AnsibleModule
 
-from kubernetes import config
-from kubernetes.config.config_exception import ConfigException
+# from kubernetes import config
+# from kubernetes.config.config_exception import ConfigException
 from openshift import client
 
+#: Regular expression for grabbing version
+VERSION_RX = re.compile("V\d((alpha|beta)\d)?")
```

Comment:
Correct. It also lets other things use the same regex by importing them instead of recreating them.